landlab/components/groundwater/dupuit_percolator.py

```diff
@@ -269,7 +269,8 @@ def __init__(
         porosity=0.2,
         recharge_rate=1.0e-8,
         regularization_f=1e-2,
-        courant_coefficient=0.01,
+        courant_coefficient=0.5,
+        vn_coefficient = 0.8,
     ):
```

Comment:
I think you will need to add something to the docstring indicating what this is.